eslint/config.js

```diff
@@ -0,0 +1,171 @@
+module.exports = {
+	parser: 'babel-eslint',
+	extends: [
+		'wordpress',
+		'plugin:wordpress/esnext',
+		'plugin:react/recommended',
+		'plugin:jsx-a11y/recommended',
+	],
+	env: {
+		browser: false,
+		es6: true,
+		node: true,
+	},
+	parserOptions: {
+		sourceType: 'module',
+		ecmaFeatures: {
+			jsx: true,
+		},
+	},
+	globals: {
+		wp: true,
+		window: true,
+		document: true,
+	},
+	plugins: [
+		'wordpress',
+		'react',
+		'jsx-a11y',
+	],
+	settings: {
+		react: {
+			pragma: 'wp',
```

Comment:
this should cover usages of `wp.createElement` etc for react components in Gutenberg.